SpatialGDK/Source/SpatialGDK/Private/Interop/ActorSystem.cpp

```diff
@@ -1694,6 +1696,44 @@ USpatialActorChannel* ActorSystem::TryRestoreActorChannelForStablyNamedActor(AAc
 	return Channel;
 }
 
+void ActorSystem::InvokeRepNotifies()
+{
+	for (FObjectRepNotifies ObjectRepNotifies : RepNotifiesToSend)
```

Comment:
Are you sure you want to make copies here instead of iterating over `const&`

Reply:
whoops - nice catch. done!